docs/development/index.rst

```diff
@@ -1,11 +1,32 @@
 .. _development:
 
 Development
-===========
+===========================
 
-.. toctree::
-   :maxdepth: 4
-   :titlesonly:
-   :glob:
+Installation
+--------------------
 
-   dev
+PyPI
+~~~~~~~~~~~~~~~~~~~~
+
+You can set up a development environment using PyPI.
```

Comment:
```suggestion
You can set up a development environment using pip.
```

This is not using PyPY, since it uses ".", meaning "here". The requirements will come from PyPY, of course, but you should probably refer to pip instead of PyPY since it is used to install the main package from a local source.